wsi/drm/drmbuffer.cpp

```diff
@@ -77,29 +77,36 @@ DrmBuffer::~DrmBuffer() {
 #endif
 }
 
-void DrmBuffer::Initialize(const HwcBuffer& bo) {
-  width_ = bo.width_;
-  height_ = bo.height_;
+void DrmBuffer::SaveMetaData(const HwcBuffer& bo) {
+  image_.handle_->meta_data_.width_ = bo.width_;
```

Comment:
hmm do we really need to do this ?

in Initialize(image_.handle_->meta_data_);  it's the same metadata struct right ? we can avoid this whole copy. ?

Reply:
We discussed this briefly and I thought you said the image_.handle_.metadata doesn't get populated during init but until it's imported.  I will remove SaveMetaData.